### ISSUE TITLE
Inaccurate Documentation of Chief Maintainer

### ISSUE THREAD

**Issue by dlstadther** [spotify/luigi#1526]
The Readme.md/rst still notes that the chief maintainer is still @Tarrasch . I have still been a bit confused about the ownership of Luigi. It seems that @erikbern is still the chief maintainer as he is the primary reviewer and merger, although Spotify split the role into 4.

During this slow transition of luigi ownership (in terms of management), should this line be removed or appended, etc.?

I know this is a super minor thing. I can see a world where new luigi developers see that Arash is chief maintainer, but not understand why his activity is less than Erik's. As active as Arash was during his reign as chief maintainer, there's no reason to diminish that reputation.

**Comment by erikbern** [comment]
I used to do most of the maintenance until end of 2014 when I left Spotify. Arash took over during 2015 and did a LOT of work. I think the last few months, Arash has been more busy, and I've been picking up some more stuff, but the truth is neither of us has a ton of time. Pretty sure Arash still spends more time than me, but the difference is not as big as during 2015

![image](https://cloud.githubusercontent.com/assets/1027979/12620555/b730b544-c4ea-11e5-82ae-448e02db81a5.png)

**Comment by Tarrasch** [comment]
I think both Erik and I do merging maintainership since the last few months since I left Spotify. Erik and I are still in touch with the up and coming maintainers though but like most people they are also busy a lot. ^^

**Comment by erikbern** [comment]
Btw this is a good time to mention – I would love it if there's an external person interested in joining as an official maintainer :)

**Comment by dlstadther** [comment]
@erikbern I may take you up on that in the future. I feel I must first gain more experience with Luigi, as well as first complete the few major projects at work that are on my plate.

**Comment by erikbern** [comment]
nice!

**Comment by dlstadther** [comment]
Revisiting this again... **What is/are the role/responsibilities of the maintainers?**

### Current Technical State

I've never maintained a project, and quite honestly Luigi is the first major project to which I've contributed. I feel that I know how to use it for my cases (although it may not be the intended Luigi pattern), which means I have pretty good knowledge of general execution (in terms of writing jobs) and modules like S3, Redshift, FTP, etc. (which are largely within contrib). The areas of Luigi I feel weak are more around the core functionality (details of the process of execution, including the central scheduler), the visualizer (and javascript in general), and testing (not necessarily Luigi-specific, just inexperienced at writing tests). Also, the relationship between Luigi, travis-ci, tox, readthedocs, and pypi are understood on a macro level (i.e. i understand the role each piece plays), but are unfamiliar to me on terms of implementation.

I have enjoyed contributing and helping others in the community. So long as I'm in my current position (which is the for the foreseeable future), I'll be working with Luigi on a more-or-less daily basis.

### Goal

While I'm here, I'd like to learn as much as I can, while also refining and expanding this project.

**Comment by Tarrasch** [comment]
@ulzha can you step up?

@dlstadther, are you volunteering to be a new maintainer? Sounds legit. :)

I think currently we just use the term "maintainer" for anyone with write-access. You can still be the most active contributor without having write access. ^^

**Comment by dlstadther** [comment]
@Tarrasch , I'm looking to put myself in a position to learn as much as I can (Luigi-wise, Python-wise, development-wise, and maintenance-wise). If I can achieve those goals by becoming a _maintainer_, then yes I'm volunteering.

**Comment by Tarrasch** [comment]
That's great news. I think you should start from whatever end you're the most comfortable with. Either you can start following and sharing thoughts in more PRs. If you want to work on some particular "project" I have quite a lot of ideas, or come with your own.

**Comment by dlstadther** [comment]
I've already changed my settings to be notified of anything posted within Luigi. For now, I'm reading/reviewing and commenting when I'm not lost. I have a few small things I'd like to add to s3.py and of course that ftp.py refactor. But at the moment, that's all I've got on luigi-radar. My focus at work will be shifted for the upcoming weeks, but as time allows I'll be participating as much as I can.
